package.json

```diff
@@ -47,13 +47,9 @@
     ]
   },
   "dependencies": {
-    "@types/express": "^4.17.6",
```

Comment:
types are dev dependencies I think?

Reply:
Not necessarily for consumable libraries. If we’re using express types anywhere in public signatures then it should be in deps

Reply:
Should upstream include the desired types explicitly though?